docs/blockchain-basics/introduction.md

```diff
@@ -15,6 +15,42 @@ import NotificationBar from '../../src/components/docs/NotificationBar';
+
+## A brief history of the "Blockchain"
+If the newer systems rely directly on Bitcoin or implement their own similar mechanics anyway, Bitcoin itself relies on a lot of pretty old information technologies (IT), from as far as 1973, and until the Genesis Block from 2009. Bitcoin's system is actually around 36 years of research, trials and experiments, and failures.
+
+Needless to say Bitcoin relies on Internet, itself relying on _TCP/IP_ from 1974. But Bitcoin and newer systems also use _Cryptography_, which is, actually, as old as Ancient History. However, what we mean by "Cryptography" in these systems is _Modern Cryptography_, which is linked to first calculating machines and research in modern Mathematics. Very important researchs like for the Diffie, Hellman, and Merkle protocol, or RSA (_Rivest, Shamir, Adleman_) and Public-Key Cryptography, are both respectively from 1976 and 1977.
+
+From there, countless reflexions on ethics, researchs on technologies, and of course Cryptography take place. Mails traceability (1981); Elliptic Curve Cryptography (1985); Digicash (1989); PGP (1991); CyberCash (1994); E-gold (1996); HashCash (1997, with Proof-of-Work); Smart Contracts (1997, defined by Nick Szabo). Even _Bit Gold_ and _B-money_ (from Wei Dai) are from 1998, and the latter already used a decentralized database recording exchanges and using an older version of the Bitcoin's _Proof-of-Work_ (more details in next chapters on this matter).
+
+In 2001, the Bittorent client gave more support for the Peer-2-Peer network trials and researchs, opening the way for more distribution and the decentralisation looked for.
+
+Finally, before Lehman bankruptcy and the financial crisis (2008), Hal Finney (1956-2014), developper on PGP (1991), released his "_Reusable Proof-of-Work_" (2004), before being one of the first exchanging bitcoins with Satoshi Nakamoto.
+
+From 2009, many projects copy Bitcoin, or try to enhance its capabilities: In 2011, Vitalik Buterin discovers Bitcoin and two years later (October, 2013), works on [a proposal for _Mastercoin_](https://web.archive.org/web/20150627031414/http://vbuterin.com/ultimatescripting.html), that would have upgrade the programmable side of the system, compared to Bitcoin, because bitcoins exchanges have scripts, embedded in _transactions_, that set the way funds can be spent. The idea is going beyond the main use-case of a currency.
+
+But the Mastercoin team didn't want to go in this direction, so Vitalik began to re-work his proposal into another project named _Ethereum_ (December, 2013). The need for even more complex scripts was driven by the interest into _**D**ecentralized **A**utonomous **O**rganizations_ (DAO), that other systems couldn't allow. That is also around December, 2013, that Gavin Wood (protocol-side) and Jeffrey Wilcke (client-side) join Vitalik Buterin to work on Ethereum. The white paper is published, and in January, 2014, Ethereum is announced.
```

Comment:
Many dates and chronological events. Would be great to have a graphic timeline from figma!